Ports/fribidi/patches/ReadMe.md

```diff
@@ -0,0 +1,23 @@
+# SerenityOS patches for fribidi
+
+## patches/0001-meson-remove-ansi-for-serenity.patch
+
+The following lines were removed from `meson.build` [(GitHub)](https://github.com/fribidi/fribidi/blob/5b9a242cbbb0cf27d20da9941667abfc63808c19/meson.build#L24-L26):
+
+```
+-if cc.get_id() == 'gcc' and cc.has_argument('-ansi')
+-  add_project_arguments('-ansi', language: 'c')
+-endif
+```
+
+The inline keyword (C99) is too new for ANSI/C89.
+
+Without this patch, the compiler freaks out because fribidi
+relies on inline functions.
```

Comment:
Please put these three lines into the commit description (of the commit that this patch is based on) and regenerate the patch and README.